app/backend/src/couchers/servicers/auth.py

```diff
@@ -237,8 +243,6 @@ def SignupFlow(self, request, context):
 
             # finish the signup if done
             if flow.is_completed:
-                # TODO: storing feedback forms
```

Comment:
Why removal of todo? Leftover and unrelated?

Reply:
Yeah, leftover and unrelated